static/charts.js

```diff
@@ -0,0 +1,89 @@
+var curChart;
+
+function setChart(labels, openedData, closedData, type) {
+    var ctx = document.getElementById("canvas");
+
+    curChart = new Chart(ctx, {  // Ignore JSHintBear
```

Comment:
You should be able to add `Chart` and `$` to the globals by adding this to the top of the file:

```
/* globals $, Chart */
```

Reply:
Thanks, worked like a charm. :)